srp_daemon/srp_daemon.service.in

```diff
@@ -9,6 +9,15 @@ Before=remote-fs-pre.target
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=@CMAKE_INSTALL_LIBEXECDIR@/srp_daemon/start_on_all_ports
+MemoryDenyWriteExecute=yes
+PrivateNetwork=yes
+PrivateTmp=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectKernelModules=yes
+ProtectSystem=full
+RestrictRealtime=yes
+SystemCallFilter=~@clock @cpu-emulation @debug @keyring @module @mount @obsolete @raw-io
 
```

Comment:
This is probably not necessary since start_on_all_ports is not network facing. I would worry that someday the calls to systemctl will break .. Other stuff looks great